app/workers/synchronizer_check_worker.rb

```diff
@@ -0,0 +1,16 @@
+class SynchronizerCheckWorker
+  include Sidekiq::Worker
+
+  RECENCY_THRESHOLD = 5 # days ago
+
+  def perform(check_since = RECENCY_THRESHOLD)
+    latest_qbe = QuotaBalanceEvent::Operation.order_by(Sequel.desc(:oid)).first
+
+    unless latest_qbe && latest_qbe.created_at >= check_since.days.ago
+      msg = "Potential sync problem on #{TradeTariffBackend.service&.upcase} service - last update: #{latest_qbe&.created_at}"
```

Comment:
The message is generic. I wonder if we could help with more insight. But probably we can't say extract much info from here.